packages/hash/frontend/src/pages/[account-slug]/types/entity-type/multiple-values-cell.tsx

```diff
@@ -0,0 +1,162 @@
+import { TextField } from "@hashintel/hash-design-system/text-field";
+import {
+  Box,
+  Checkbox,
+  ClickAwayListener,
+  Collapse,
+  Fade,
+  Popper,
+  TableCell,
+  Typography,
+} from "@mui/material";
+import { useEffect, useState } from "react";
+import { useFormContext, useWatch } from "react-hook-form";
+import { EntityTypeEditorForm } from "./form-types";
+
+export const MultipleValuesCell = ({
+  propertyIndex,
+}: {
+  propertyIndex: number;
+}) => {
+  const { register, setValue } = useFormContext<EntityTypeEditorForm>();
+
+  const [anchorEl, setAnchorEl] = useState<HTMLDivElement | null>(null);
+  const [multipleValuesMenuOpen, setMultipleValuesMenuOpen] = useState(false);
+
+  const [array, minValue, maxValue] = useWatch({
+    name: [
+      `properties.${propertyIndex}.array`,
+      `properties.${propertyIndex}.minValue`,
+      `properties.${propertyIndex}.maxValue`,
+    ],
+  });
+
+  useEffect(() => {
+    if (minValue > maxValue) {
+      setValue(`properties.${propertyIndex}.maxValue`, minValue);
+    }
+    // Should only be triggered when minValue updates
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [propertyIndex, setValue, minValue]);
+
+  useEffect(() => {
+    if (maxValue < minValue) {
+      setValue(`properties.${propertyIndex}.minValue`, maxValue);
+    }
+    // Should only be triggered when maxValue updates
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [propertyIndex, setValue, maxValue]);
```

Comment:
Okay, looked into this a bit. The better way to implement this would be with the `min` / `max` attributes on the inputs. The `max` attribute on the min field should be set to the value of the `max` field, and vice versa. Don't need any special code for this. Annoyingly means you have to make this component re-render when those values change, but thats fine really.

Reply:
This leads to a slightly different behavior:

Old:
![old](https://user-images.githubusercontent.com/37453800/199243211-ab89803c-21c2-46e4-b006-061bed3bc079.gif)


New: 
![new](https://user-images.githubusercontent.com/37453800/199243267-a011a52f-e3d2-49b8-a030-1d22ec1b0b20.gif)


I'm okay with either approach since none of this is contemplated in the designs, but I would like to hear @nonparibus's opinion on this.

Reply:
Option 1 please; let's retain the originally implemented functionality here.

Reply:
Understood – will follow up shortly. Apologies for not realising the implemented behaviour was desirable.

Reply:
Okay – this codesandbox explains how to do this without requiring `watch` or `useEffect`. 

Generally I wish there was a way to describe this behaviour declaratively at the form level, rather than at the input level, but doesn't seem to be a good way to describe this.

If we're going to do this at the component/input level, custom on change handlers with `getValues`/`setValue` seem to be the cleanest.

https://codesandbox.io/s/min-max-react-hook-form-eeb3lh?file=/src/index.jsx

Reply:
Thank you for the cleaner approach (edf077b5cf4b17fc8811877f1abb6f630709fc5f).